README.md

```diff
@@ -35,25 +35,38 @@ Sugarcoat was created to enable developers to produce rich UI documentation easi
 
 # Features #
 
-1. What does it look like?
+1. You decide where your library lives
 
-   A [Sample Site](https://sapientnitrola.github.io/sugarcoat/) and its accompanying [Pattern Library](https://sapientnitrola.github.io/sugarcoat/documentation/index.html). These pages can be found on your local copy by following a short series of steps: [Example Library](#example-library).
+  Sugarcoat will never impose a folder structure on you.
 
-2. Can you parse css-preprocessor variables?
+2. No extra work
 
-   Yes, we're able to grab the variables from less, SASS, or regular CSS files. Just set the `type` option to `variable` in the appropriate `section` object within the `sections` array. See [`section.type`](#type) for more information.
+  You don't have to create any extra files outside of your files for your pattern library. Just create your config and document your project files as you normally would.
 
-3. What comment structure do I need to use?
+3. Universal Comment Styles
 
-   You can customize what delimiters Sugarcoat uses. By default, Sugarcoat will parse a JSDoc-style comment block `/** ... */`: [Code Comment Syntax](#code-comment-syntax).
+  Use the same JSDoc commenting syntax across all file types. Just be sure to wrap your comments in a standard HTML comment block in your html files. And if you've got inline comments, Sugarcoat will ignore them.
 
-4. Can I customize the default template that comes with Sugarcoat?
-   
-   Absolutely. The `template` option in the [`settings` Object](#settings-object) enables you to define your own layout, partials and static assets. Once those are set, you can use the [`template`](#template) option in the [`section` Object](#section-object) which allows you to override the partial for a particular section.
+4. Easy-to-identify component states
 
-5. How do you handle style bleed?
+  You can tell sugarcoat that there are modifier states in your css, right within your comment block! Sugarcoat will help highlight and display them for extra readability.
+
+5. Variables Galore
+
+  Sugarcoat will still understand your variables if they're SCSS, LESS, or even future specs: `--my-var`.
+
+6. No Style Bleed
+
+  The styles that come out of the box with Sugarcoat will not allow for any bleeding of styles into your components or modules. For now, you'll need to prefix all of your own Sugarcoat style selectors with `.sugar-example`. It's on the roadmap to have Sugarcoat prefix your files for you based on a default or custom selector, and insert your stylesheets into the head of your generated pattern library accordingly. *Note: We chose not to use iframes because we didn't want to resize different iframes as you interacted with a component with varying height (such as a custom dropdown).*
+
+7. No weird comment deliminters
```

Comment:
I disagree. 3 has to do with the comments that sugarcoat uses to pick up the users comments. Where 6 has to do with how Sugarcoat index page styles will not bleed into your components (and their styles).